react-app/src/components/projects/AddProject.tsx

```diff
@@ -201,23 +202,26 @@ const DisposalProjectSearch = (props: IDisposalProjectSearch) => {
 };
 
 const AddProject = () => {
+  const navigate = useNavigate();
   const formMethods = useForm({
     defaultValues: {
       Name: '',
       TierLevelId: null,
       Notes: '',
-      Assessed: '',
-      NetBook: '',
-      Estimated: '',
-      Appraised: '',
+      Assessed: undefined,
```

Comment:
Setting these to undefined will throw errors uncontrolled -> controlled errors in the MUI components.

Reply:
So 0's then? Numbers were just expected on the other side.

Reply:
Replaced with 0s.